packages/mobile/src/web3/saga.ts

```diff
@@ -267,6 +224,9 @@ export function* watchRefreshGasPrice() {
 }
 
 export function* web3Saga() {
-  yield spawn(checkSyncProgressWorker)
```

Comment:
Shouldn't `getConnectedAccount` use the new web3 wait? Many functions rely on that helper

Reply:
done